GameDefinition/Content/ModeKind.hs

```diff
@@ -264,7 +283,11 @@ ambush = ModeKind  -- dense ranged with reaction fire vs melee at night
   , mdesc   = "Prevent hijacking of your ideas at all cost! Be stealthy, be observant, be aggressive. Fast execution is what makes or breaks a creative team."
   , mreason = "In this adventure, finally, your heroes are able to all use ranged attacks at once, given enough ammunition."
   , mhint   = "Beware of friendly fire, particularly from explosives. But you need no more hints. Go fulfill your destiny! For Science!"
-  }
+  } where
```

Comment:
I think the formatting with `where` on a separate one that you used before, is better.